containers/integration-tests-build/Dockerfile

```diff
@@ -23,3 +23,20 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
         az-dcap-client \
         libsgx-quote-ex \
         libsgx-dcap-ql
+
+
+# Install Docker inside integration-tests container, so it can spin another docker container to build enclaves
+RUN apt-get update
+RUN apt-get install -y \
```

Comment:
Can you merge this with the `install` command at the top.

Reply:
Done.